Expanded diversity and inclusion metrics.

BenLloydPearson opened the pull request:
I expanded these metrics in a similar fashion as was done to the Growth, Maturity, and Decline metrics. All of the items that were originally listed here are still here except for one that I thought was redundant. 

I also added a few new metrics under organizational diversity, please review them to ensure they are clear.

GeorgLink commented:
@BenLloydPearson sorry for letting this sit for so long. We moved the Diversity and Inclusion work into a new workgroup and should discuss these changes there.

germonprez commented:
Hey @BenLloydPearson not sure how I missed this one too. Sorry! I'll check it out.

GeorgLink commented on 1_Diversity-Inclusion.md:
```diff
@@ -1,29 +1,53 @@
+Contributors per Organization | What is the number of individual contributors from each organization?
+Maintainer Organizational Representation | What is the ratio and number of maintainers from each organization? 
+Maintainers per Organization | What is the ratio and number of maintainers from each organization? 
+Maintainer Commit Organizational Diversity | What is the number of commits a maintainer makes for contributors from each organization?
```
What does this question get at? 

- What is, for each organization, the commit ratio for maintainers versus contributors from the same organization?

GeorgLink commented on 1_Diversity-Inclusion.md:
```diff
@@ -1,29 +1,52 @@
+Contributions per Organization | What is the number of contributions from each organization?
+Contributors per Organization | What is the number of individual contributors from each organization?
+Maintainers per Organization | What is the ratio and number of maintainers from each organization? 
+Maintainer Commit Organizational Diversity | What is the number of commits a maintainer makes for contributors from each organization?
```
What does this question get at? (reposting question)

* What is the ratio of commits from an organization's maintainer compared to the number of contributors from the same organization?
* What is, for each organization, the commit ratio for maintainers versus contributors from the same organization?
* What is the average of the commit ratios for maintainers versus contributors from the same organization?

germonprez replied:
Which question? All four of them?

GeorgLink replied:
Line 25

germonprez replied:
I'm not really sure either.

BenLloydPearson commented:
I added that line back in and fixed the description. Line 24 describes the number of maintainers each organization has, line 25 describes the number of organizations that are represented by one or more maintainers.

GeorgLink commented:
Thanks for the clarification @BenLloydPearson 

What does this question (line 26: Maintainer Commit Organizational Diversity) get at?

*    What is the ratio of commits from an organization's maintainer compared to the number of contributors from the same organization?
*    What is, for each organization, the commit ratio for maintainers versus contributors from the same organization?
*    What is the average of the commit ratios for maintainers versus contributors from the same organization?

BenLloydPearson commented:
@GeorgLink This one is a bit confusing, but I'll try my best.

Essentially, for the commits made by each maintainer, what organizations do the authors of the patches work for?

I.e. if a maintainer commits 10 patches in total, 5 of them might be their own, and 5 might be from an author that works for another company. This means they'd have 5 commits for their own organization, and 5 commits for another organization.

GeorgLink commented:
Thanks @BenLloydPearson.
That makes sense now. 
How about:

* What is the ratio of patches a maintainer accepts that are authored by contributors from other organizations than the one the maintainer works at?

BenLloydPearson commented:
@GeorgLink 

I'm not sure that quite hits what I'm trying to illustrate, particularly because your version specifically focuses only on commits from other organizations, where I think commits from the maintainer's organization are equally important. E.g. if a maintainer makes 100% of their commits on behalf of their company, I think that's something important to look at.

How does this sound:  What is the ratio and number of commits from each maintainer for all authors grouped by organization.

GeorgLink commented:
That wording makes sense to me. Thanks @BenLloydPearson 
I updated the file. 

Please review and approve.

BenLloydPearson commented:
Looks great to me. I can't approve it since I made the PR.

GeorgLink commented:
Thanks @BenLloydPearson 
I'm merging this now, since we had enough time for feedback and have addressed all questions.

rpaik commented:
I'm late to the party (as I'm back from 2 weeks of conferences)....  I like the new additions, however, I also have a question on "maintainer commit organizational diversity".  Do we also want to look at organizational diversity for all committs (not just from maintainers)?  I ask as it'd also be interesting to look at commits from contributors (who are not maintainers yet)...

GeorgLink commented:
Welcome Back @rpaik 
To add to your question: We discussed also diversity of reviews: #49

rpaik commented:
I think I was confused with the terminology.  I just realized commit here means merging code (which only maintainers can do).  So I'm good.  Thanks @GeorgLink
